routes/socket/routes.js

```diff
@@ -530,6 +531,17 @@ module.exports.socketRoutes = () => {
 					}
 				}
 			});
+			socket.on('modGetRemakes', data => {
+				const uid = data.uid;

```

Comment:
If `data` is `null` (which it can be), this causes a crash. So return early if `data` or `game` is null.